internal/version/version.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 Tetrate
```

Comment:
so now what's the criteria for deciding whether a package should sit in interna/ or pkg/?

Reply:
over time, everything except main.go will be internal, as we don't have a public api